docs/authoring-components.md

```diff
@@ -677,31 +679,45 @@ the aforementioned rule, we would also like to provide lint rules to enforce the
   could potentially be localized, etc.) must be referenced by a `strings` static getter.
 - All semantic numbers leveraged by the adapter (timeout lengths, transition durations, etc.) must
```

Comment:
Why do numbers need to be leveraged by the adapter? I understand why you need the class names for CSS renaming, but why do we need DEFAULT_THROTTLE_DELAY_MS exposed?